### ISSUE TITLE
Do we have version for 14.0.0.0?

### ISSUE THREAD

**Issue by mamoyko** [Strackeror/MHW-QuestLoader#6]
Do we have version for 14.0.0.0?

**Comment by Giuliadilsa** [comment]
Yup, my fork: https://github.com/Giuliadilsa/MHW-QuestLoader-14.0.0-Quest-Loading-Fix

**Comment by mamoyko** [comment]
thanks man

**Comment by rubenmaster** [comment]
@Giuliadilsa 

Hello
But I have doubts about the installation or correct use of this.
I was able to explain to me how to install / use it.
Thanks

**Comment by mystown** [comment]
@rubenmaster Sorry but Im getting trouble installing stracker's loader for 14.0.0. Can you tell me how ? I have clicked the link and it shows multiple different files like the image I show below. If you can, please explain which file I need to install and how to install them.
![image_2023-04-16_113746532](https://user-images.githubusercontent.com/130921014/232327168-c1cb909b-6d96-47af-bbf5-80ead9ad634c.png)
